.github/workflows/deploy-enterprise.yml

```
@@ -80,7 +80,7 @@ jobs:
             image_tag: latest
             product_lifetime_in_days: 365000
             quota: 0
-            job_files: job_spec/INSAR_GAMMA.yml
+            job_files: job_spec/AVO.yml
```

```suggestion
            job_files: job_spec/INSAR_GAMMA.yml job_spec/INSAR_GAMMA_TEST.yml
```
We don't want to take away the production INSAR_GAMMA job type